src/CodeStyle/CSharp/Analyzers/Microsoft.CodeAnalysis.CSharp.CodeStyle.csproj

```diff
@@ -16,8 +14,6 @@
   <ItemGroup Label="Project References">
     <ProjectReference Include="..\..\Core\Analyzers\Microsoft.CodeAnalysis.CodeStyle.csproj" />
   </ItemGroup>
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|AnyCPU'" />
```

Comment:
The old project system actually relied on this precise pattern to know what configurations exist and deleting them actually broke the IDE. Is CPS the same way? @davkean or @tmeschter?

Reply:
CPS does not use these when using the new configuration syntax.